website/content/en/docs/advanced-topics/scorecard/kuttl-tests.md

```diff
@@ -4,160 +4,224 @@ weight: 50
+configuration includes a single test directory `v1alpha1`, the path to our test steps relative
+to the config file. The `# +kubebuilder:scaffold:kuttl:testDirs` comment helps `operator-sdk`
+commands update this file when new APIs are added, so you don't need to worry (or remove) it.
+
+Next, if you peek inside `01-assert-memcached.yaml` you'll see something like the following:
+
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-test
+spec:
+  foo: bar
 ```
-.
+
+Yep, it's a plain Memcached CR manifest, and it _is_ `kubectl apply`-able! Most if not all kuttl
+test case files are just Kubernetes YAML manifests that are either merge-patched against in-cluster
+objects or compared in assert or error scenarios. Hopefully you can see how powerful this
+declarative testing pattern can be.
+
+For further reading take a look through the kuttl [testing docs][kuttl-testing].
+
+## Running kuttl Test Suites with `scorecard`
+
+`operator-sdk scorecard` runs your kuttl suite by passing it to `kubectl-kuttl`, the kuttl testing CLI tool,
+from within the `quay.io/operator-framework/scorecard-test-kuttl` image. Since `scorecard` runs tests against
+a running operator using its [bundle][scorecard-bundle], your kuttl tests are expected to exist within either a
+[local `bundle/` directory][doc-bundle] generated using `make bundle` or a bundle image. Typically you
+should not bundle test dependencies with your production manifests, so it is currently recommended
+that you simply copy your kuttl manifests into `bundle/` while testing.
+
+Lets walk through configuring kuttl to run with scorecard, assuming your operator is running in-cluster already.
+Initially your bundle may look like the following:
+
+```console
+$ tree bundle/
+bundle/
 ├── manifests
-│   ├── cache.example.com_memcacheds_crd.yaml
+│   |   ...
 │   └── memcached-operator.clusterserviceversion.yaml
 ├── metadata
 │   └── annotations.yaml
 └── tests
     └── scorecard
-        ├── config.yaml
-        └── kuttl
-            ├── kuttl-test.yaml
-            └── list-pods
-                ├── 00-assert.yaml
-                └── 00-pod.yaml
-            └── list-other
-                ├── 00-assert.yaml
-                └── 00-pod.yaml
+        └── config.yaml
 ```
 
- * `bundle/` - Contains bundle manifests and metadata under test.
- * `bundle/tests/scorecard/config.yaml` - Configuration yaml to define and run scorecard tests.
- * `tests/kuttl` - Contains tests written for kuttl to execute
- * `tests/kuttl/kuttl-test.yaml` - Contains the kuttl configuration, it is here that you would add any kuttl specific configuration settings that you might require.
- * `tests/kuttl/list-pods` - Contains a kuttl test case
- * `tests/kuttl/list-pods/00-assert.yaml` - Contains a kuttl test case assert
- * `tests/kuttl/list-pods/00-pod.yaml` - Contains a kuttl test case step
- * `tests/kuttl/list-other` - Contains another kuttl test case
```

Comment:
why the above desc was removed?

Reply:
These files are discussed at the top of the doc now.